.eslintrc.json

```diff
@@ -1,14 +1,18 @@
 {
-  "extends": ["airbnb", "prettier", "prettier/react"],
+  "extends": ["airbnb", "prettier"],
   "plugins": ["prettier"],
   "env": {
     "jest": true
   },
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
   "rules": {
     "strict": 0,
     "import/no-extraneous-dependencies": 0,
     "react/forbid-prop-types": 0,
+    "react/function-component-definition": 0,
+    "react/jsx-props-no-spreading": 0,
+    "prefer-object-spread": 0,
+    "class-methods-use-this": 0,
```

Comment:
I've been mulling this rule over and see that there is no need to turn it on as class components are deprecated and we won't have future implementations of class components. If the components already work perfectly, I see no need to refactor the code to meet this rule.